kernel/packages/shared/loading/ReportFatalError.ts

```diff
@@ -110,18 +110,19 @@ export async function ReportFatalErrorWithUnityPayloadAsync(error: Error, contex
 }
 
 export function ReportFatalError(error: Error, context: ErrorContextTypes, payload: Record<string, any> = {}) {
-  const finalPayload = GetErrorPayload(context, payload)
   const eventData = {
     // attach every field from the payload to the event
-    ...(finalPayload || {}),
+    ...(payload || {}),
+    // set the context property
+    context,
     // this is on purpose, if error is not an actual Error, it has no message, so we use the ''+error to call a
     // toString, we do that because it may be also null. and (null).toString() is invalid, but ''+null works perfectly
     message: error.message || '' + error,
     stack: getStack(error).slice(0, 10000)
   }
 
   trackEvent('error_fatal', eventData)
-  ReportRollbarError(error, finalPayload)
+  ReportRollbarError(error, eventData)
```

Comment:
We shouldn't send the `eventData` like this because the stack and message fields are redundant, rollbar already brings it from the error object.